packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1381,6 +1383,8 @@ export class ContainerRuntime
 			snapshotWithContents,
 		} = context;
 
+		this._context = context;
```

Comment:
I don't think we should add `_context` in this PR.  Instead, follow the example of `_getClientId` and have a new private member that you set to a function like `() => context.connected`.

I'm not entirely sure why we don't just stash the Context directly on this class, but for now let's follow the existing pattern.